src/source/examples/get-campaign-vendor-id-v5-example.rst

```diff
@@ -0,0 +1,27 @@
+.. code-block:: javascript
```

Comment:
this already should exist in the yaml, worth double checking if it's correct though